src/arm/tables.S

```diff
@@ -41,7 +41,6 @@
 .global X(mc_subpel_filters)
 .hidden X(mc_subpel_filters)
 const X(mc_subpel_filters), align=3
-X(mc_subpel_filters):
 .byte  0,   1,  -3,  63,   4,  -1,   0,   0 /* REGULAR */
```

Comment:
For the commit message of this one, I'd write the last section like this:

"The const macro implicitly defines a label, so the manually defined ones are duplicates."